caddyhttp/httpserver/plugin.go

```diff
@@ -490,6 +490,8 @@ var directives = []string{
 	"hugo",      // github.com/hacdias/caddy-hugo
 	"mailout",   // github.com/SchumacherFM/mailout
 	"awslambda", // github.com/coopernurse/caddy-awslambda
+
```

Comment:
What is the best way to determine where in the chain are the best place to put the directive?